atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -113,6 +116,10 @@ static void waitForSchemaVersions(
         }
     }
 
+    private static int getNumberOfNodes(Map<String, List<String>> versions) {
```

Comment:
`getNumberOfVisibleNodes` 

After all, we now know that this may not be the number of nodes in the cluster.